app/packages/multi_trigger_autocomplete/lib/src/autocomplete_controller.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+
+class AutocompleteTextController extends TextEditingController {
```

Comment:
This is the main class which extends ```TextEditingController``` to allow us build custom text spans based on trigger characters. ```triggerStyles``` is passed down here from main widget ```MultiTriggerAutoComplete```.